cosmoz-image-viewer.js

```diff
@@ -5,199 +5,67 @@
 
 	Polymer({
 		behaviors: [
-			Polymer.IronResizableBehavior,
 			Cosmoz.TemplateHelperBehavior,
 			Cosmoz.TranslatableBehavior
 		],
 		is: 'cosmoz-image-viewer',
 		properties: {
-			currentImage: {
-				type: Object,
-				observer: '_currentImageChanged'
-			},
-			currentImageIndex: {
-				type: Number,
-				computed: '_computeCurrentImageIndex(currentImage, images)'
-			},
-			currentPage: {
-				type: Number,
-				computed: '_computePage(currentImageIndex)'
-			},
-			detached: {
```

Comment:
This is not part of PhotoSwipe, it's a feature to detach the image container (also not part of photoswipe) into a separate window.